### PR TITLE
Prevent admins from accessing standalone projects.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Alert (after redirect):

image

### CODE REVIEW

File: dashboard/test/controllers/projects_controller_test.rb [hunk 1]
```diff
@@ -180,10 +180,21 @@ class ProjectsControllerTest < ActionController::TestCase
...
+
+    %w(applab gamelab).each do |lab|
+      get :load, params: {key: lab}
+      assert_redirected_to '/'
+    end
+
+    %w(applab gamelab).each do |lab|
+      get :show, params: {key: lab, share: true, channel_id: 'fake_channel_id'}
+      assert_redirected_to '/'
+    end
```
Comment (aoby) [inline dashboard/test/controllers/projects_controller_test.rb:197]: Do we care to assert the flash alert too?

Comment (ashercodeorg) [reply dashboard/test/controllers/projects_controller_test.rb:197]: Not really (at least not enough to change it). The important thing is the user (admin) is not able to access the standalone project.

**Comment by davidsbailey** [comment]
Heads up that this broke Josh Schulte's ability to unblock abusive projects. See https://codedotorg.slack.com/archives/C0T0UQPLZ/p1494612720686467 for workaround. Probably a UI for this should be added to https://studio.code.org/admin

**Comment by ashercodeorg** [comment]
Roger, I'll own doing so. Thanks for notifying me of this, though I'll call this working as intended.

**Comment by davidsbailey** [comment]
Awesome, thanks for taking this work on, Asher !

**Pull request merged**